dist/pkg/PKGBUILD.in

```diff
@@ -0,0 +1,32 @@
+# Maintainer: Christopher Hoage <iam@chrishoage.com>
+
+pkgname=ipmi-fan-control
+pkgver=@VERSION@
+pkgrel=1
+pkgdesc="SuperMicro IPMI fan control daemon"
+url="https://github.com/chenxiaolong/$pkgname"
+source=("@TARBALL_NAME@" "ipmi-fan-control.service")
```

Comment:
I'd suggest maybe excluding the systemd service from `source=(...)` since it's already included in the tarball. Then you could have:

```bash
package() {
  cd "$srcdir/$pkgname-$pkgver"

  install -m755 -d "${pkgdir}/usr/lib/systemd/system/"
  sed \
    -e "s/@BINDIR@/\/usr\/bin/g" \
    -e "s/@SYSCONFDIR@/\/etc/g" \
    < "dist/${pkgname}.service.in" \
    > "${pkgdir}/usr/lib/systemd/system/${pkgname}.service"

  install -Dm755 "target/release/$pkgname" "${pkgdir}/usr/bin/$pkgname"
  install -Dm 644 config.sample.toml "${pkgdir}/etc/${pkgname}.toml"
}
```

and avoid the need to preprocess the file in `build_source_package.sh`.

Reply:
Good thought, unsure why I didn't consider that myself.